cpp/modmesh/buffer/SimpleArray.hpp

```diff
@@ -32,6 +32,8 @@
 #include <modmesh/math/math.hpp>
 #include <modmesh/simd/simd.hpp>
 
+#include <modmesh/toggle/RadixTree.hpp>
```

Comment:
Include the whole `toggle` module by writing:

```cpp
#include <modmesh/toggle/toggle.hpp>`
```

It is a convention in modmesh to include the module header when you are outside the module.